### PR TITLE
updated devel to 2.3

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Feature Pull Request
##### COMPONENT NAME

version
##### ANSIBLE VERSION

```
2.3
```

### CODE REVIEW

File: ROADMAP.rst [hunk 1]
```diff
@@ -16,10 +16,11 @@ These roadmaps are the team's *best guess* roadmaps based on the Ansible team's
 =======================================
 Current 2.2 Roadmap is open for comment
 =======================================
-We are currently trying to finalize the 2.2 Roadmap, targeted for September delivery.  We are seeking community feedback for 2.2.  That document is here:  
+We are currently trying to finalize the 2.3 Roadmap, targeted for January delivery.  We are seeking community feedback for 2.3.  That document is here:
+ - `2.2 Roadmap Document <docsite/rst/roadmap/ROADMAP_2_3.rst>`_
```
Comment (mattclay) [inline ROADMAP.rst]: Accidental 2.2 reference here.

File: ROADMAP.rst [hunk 2]
```diff
@@ -16,10 +16,11 @@ These roadmaps are the team's *best guess* roadmaps based on the Ansible team's
 =======================================
 Current 2.2 Roadmap is open for comment
```
Comment (gundalow) [inline ROADMAP.rst]: 2.3

File: ROADMAP.rst (hunk 1 above)
Comment (bcoca) [reply ROADMAP.rst]: fixed

File: ROADMAP.rst (hunk 2 above)
Comment (bcoca) [reply ROADMAP.rst]: fixed

**Pull request merged**